src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -681,7 +681,7 @@ const schema = {
     secondaryPtsdIncident: {
       type: 'object',
       properties: {
-        authorities: {
+        source: {
```

Comment:
We should aim to decrease reliance on this schema file, updating pages to point at the actual schema in `vets-json-schema` instead. The goal is to eventually get rid of this file entirely. Also, I think this property is called `sources` in the vets-json-schema (https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/21-526EZ-allclaims/schema.js#L179)

Reply:
Yes, and I don't think we're referencing at anywhere at this point.  I think I'd updated this to match the schema changes and then didn't follow up when it changed further.  I can update this to match the schema if you'd like.

Reply:
I was just thinking about this myself - would make it easier to unwind later.